sklearn/neighbors/base.py

```diff
@@ -17,6 +17,7 @@
 from ..base import BaseEstimator
```

Comment:
Please remove these changes to neighbors from the PR, to help the next reviewer. Thank you for your work on them when we thought that was the right direction. We have all learnt a lot and are in a much better place now. Consider submitting as a separate PR.